core/test/TokenizedDerivative.js

```diff
@@ -42,7 +42,7 @@ contract("TokenizedDerivative", function(accounts) {
   const apDelegate = accounts[3];
 
   const name = "1x Bitcoin-Ether";
-  const symbol = "BTCETH";
+  const symbol = "UMAETHUSD";
```

Comment:
Why this unrelated change?

Reply:
I updated the comments in how the tests flow and changed some of their magic numbers away from a Synthetic `BTCETH` pair as this was too complex to reason about in the maths.